js/src/personal/kelly/Kelly.module.css

```diff
@@ -29,13 +12,13 @@
   height: 100%;
   top: 0;
   left: 0;
-  background-color: rgb(250, 236, 216); /* White with slight opacity */
-  z-index: 9999; /* Ensures it sits on top */
+  background-color: #f8e6bc;
+  z-index: 9999;
```

Comment:
nit: Generally you shouldn't go all the way to 9999, in case you want to put something on top of it.

I think it's generally good to keep a list somewhere of what z-index's you have in your code, but I feel like people slack on keeping that updated.